### PR TITLE
Bug 1801064: Add '\n' for pipeline deprecate info

### PR DESCRIPTION
**Pull request by wewang58**
/assign @adambkaplan

### CODE REVIEW

**Comment by adambkaplan** [comment]
@soltysh is there a reason the newline is not needed in https://github.com/openshift/oc/blob/master/pkg/cli/rollout/latest.go#L114 ?

File: pkg/cli/newapp/newapp.go [hunk 1]
```diff
@@ -458,7 +458,7 @@ func (o *AppOptions) RunNewApp() error {
 			}
 		case *buildv1.BuildConfig:
 			if t.Spec.Strategy.Type == buildv1.JenkinsPipelineBuildStrategyType {
-				fmt.Fprintf(o.Action.ErrOut, "JenkinsPipeline build strategy is deprecated. Use Jenkinsfiles directly on Jenkins or OpenShift Pipelines instead")
+				fmt.Fprintf(o.Action.ErrOut, "JenkinsPipeline build strategy is deprecated. Use Jenkinsfiles directly on Jenkins or OpenShift Pipelines instead\n")
```
Comment (soltysh) [inline pkg/cli/newapp/newapp.go]: Maybe just use `fmt.Fprintln` instead of manually adding the newline.

**Comment by soltysh** [comment]
> @soltysh is there a reason the newline is not needed in https://github.com/openshift/oc/blob/master/pkg/cli/rollout/latest.go#L114 ?

With `Fprintf` you're responsible for formatting, that's why I propose to use `Fprintln` my bad I didn't catch it in your PR.

File: pkg/cli/newapp/newapp.go (hunk 1 above)
Comment (wewang58) [reply pkg/cli/newapp/newapp.go]: @soltysh Updated it, thanks

**Pull request merged**